projects/components/src/load-async/load-async.directive.ts

```diff
@@ -23,8 +24,11 @@ import {
 export class LoadAsyncDirective implements OnChanges, OnDestroy {
   @Input('htLoadAsync')
   public data$?: Observable<unknown>;
+  @Input()
+  public htLoadAsyncLoaderType!: LoaderType;
+
   private readonly wrapperParamsSubject: ReplaySubject<LoadAsyncWrapperParameters> = new ReplaySubject(1);
-  private readonly wrapperInjector: Injector;
+  private wrapperInjector!: Injector;
```

Comment:
any reason for removing the readonly?

Reply:
<img width="801" alt="Screen Shot 2021-09-06 at 8 49 47 PM" src="https://user-images.githubusercontent.com/83946550/132220068-65fd9230-2219-484e-9633-eae02a014d1a.png">
its old code

Reply:
updated